docs/ref/django-admin.txt

```diff
@@ -95,8 +95,6 @@ providing a list of app labels as arguments::
 
```

Comment:
Above it already says "by default all apps will be checked", so repeating that afterwards was not necessary.